tobiasHe/assignments/twopointers/lc15/lc15.js

```diff
@@ -1,14 +1,49 @@
 // Problem Statement #
 // Given an array of unsorted numbers, find all unique triplets in it that add up to zero.
-
 function search_triplets(arr) {
-  triplets = [];
-  // TODO: Write your code here
-  return triplets;
+	// TODO: Write your code here
+	// given unsorted array, find all triplets that add up to zero, return array with all triplets that sum up to zero
+	// sort the array first
+	// start from both ends of the array
```

Comment:
It seems that this approach is incomplete, what happens after starting from both end of the array?